htdocs/class/xoopsform/renderer/XoopsFormRendererTailwind.php

```
@@ -0,0 +1,836 @@
+        $title   = $this->esc($element->getTitle(false));
+        $value   = $element->getValue();
+        $options = $element->getOptions();
+
+        $ret = '<select class="select select-bordered w-full"'
+            . ' size="' . (int) $element->getSize() . '"'
+            . $this->renderExtra($element);
+        if ($element->isMultiple() !== false) {
+            $ret .= ' name="' . $name . '[]" id="' . $name . '" title="' . $title . '" multiple="multiple">';
+        } else {
+            $ret .= ' name="' . $name . '" id="' . $name . '" title="' . $title . '">';
+        }
```

PHPMD (2.15.0)

[error] The method renderFormSelect uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)